ddtrace/contrib/openai/patch.py

```diff
@@ -175,6 +174,10 @@ def trunc(self, text):
 log = get_logger(__file__)
 
 
+def _wrap_classmethod(obj, wrapper):
+    wrap(obj.__func__, wrapper)
```

Comment:
I wonder if this should live in a tracer wrapping utility module, like the original wrapping utilities based on wrapt.